R/plotting.R

```diff
@@ -225,36 +227,50 @@ SortHeatmap <- function(slice) {
   return(slice)
 }
 
-### Make coloured annotation bars for each sample provided and their corresponding categories
+### Make a list of coloured annotation bars for each sample provided and their corresponding categories
+# ann_df - dataframe, must contain at least `sample_id` and all columns in `cols`
+# cols - vector of categorical columns to include as annotations, i.e. one column = one annotation bar
+# colors - vector of corresponding viridis color options for each column, any letter A-H
```

Comment:
This explanation is close, but perhaps an example would help.
ex. xx <- c('A', 'B'), if you have two bars
Also please provide a default (of 1 of course).